service-commands/src/commands/seed/UserCache.js

```diff
@@ -1,93 +1,97 @@
-const fs = require('fs')
+const fs = require("fs")
 
-const {
-  Constants
-} = require('../../utils')
+const { Constants } = require("../../utils")
 
-const {
-  SEED_CACHE_PATH
-} = Constants
+const { SEED_CACHE_PATH } = Constants
 
 class UserCache {
-    constructor() {
-      this.CACHE_PATH = SEED_CACHE_PATH
-    }
+  constructor() {
+    this.CACHE_PATH = SEED_CACHE_PATH
+  }
 
-    update = (cacheObject) => {
-      fs.writeFileSync(this.CACHE_PATH, JSON.stringify(cacheObject))
-      return
-    }
+  update = (cacheObject) => {
+    fs.writeFileSync(this.CACHE_PATH, JSON.stringify(cacheObject))
+    return
+  }
 
-    getActiveUser = () => {
-      const cache = this.get()
-      const activeAlias = cache['active']
-      return cache[activeAlias] || {}
-    }
+  getActiveUser = () => {
+    const cache = this.get()
+    const activeUser = Object.entries(cache).find(([alias, details]) => {
+      return details && details.active
+    })
+    return activeUser[1] || {}
```

Comment:
what is at `activeUser[1]`?

Reply:
i will clarify in code - it's the details object for user bc the key is the alias/hedgehogEntropyKey